content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -0,0 +1,94 @@
+---
+title: DiagnosticReport | Soarian DSTU 2 API
+---
+ 
+# DiagnosticReport
+
+* TOC
+{:toc}
+
+## Overview
+
+For the DiagnosticReport resource query, the API returns test names and result values for test types that the Soarian client has configured as appropriate for the API.  The ONC (Office of the National Coordinator) CCDS (common clinical data set) definition is defined to include laboratory tests and values however clients may configure additional qualifying test types such as radiology, cardiology, etc.  The configuration of qualifying test types is a single configuration that applies to both the API and the Consolidated Clinical Document Architecture (C-CDA) document available to the patients. 
+
+The API returns the test names and values in reverse chronological order by result date and, within that, the sequence that is configured in the Soarian flowsheet that defines the qualified tests.  Qualified results of status Final, Corrected, or Cannot Obtain are returned.  Result values of numeric or text are returned; image only results, including those encapsulated in non-text formats such as pdf, are not returned.  Since each result is individually dated and the value is unambiguously relevant for that date, this resource supports date-based queries.
+
+## Terminology Bindings      
+
+
+<%= terminology_table(:soarian_diagnostic_report, :dstu2) %>
+
+## Extensions
+The content of these extensions must be visible to the user, and recorded with any storage or transmit activity along with the query/response date/time, as it represents the best representation of what clinicians recorded for the patient at the time of query.  
+
+
+| Name | Condition | Description |
```

Comment:
The extension name seems wrong. I would throw this under a subcategory of Custom Extensions and adjust the table. See example here: http://fhir.cerner.com/millennium/dstu2/medications/medication-statement/#custom-extensions

In order for an app to actually use an extension, they need to know the defining URL, what type of value to expect, and what the extension is. Something that is missing from the example above is what element the extension applies to. You could add that in the description (eg: applies to Resource.text or narrative)

If these extensions are supported on all or most resources (as it appears to be), I would move them to the overview and link to them from each resource.